.changeset/config.json

```diff
@@ -26,7 +26,7 @@
   "access": "public",
   "baseBranch": "dev",
   "updateInternalDependencies": "patch",
-  "ignore": [],
+  "ignore": ["integration", "integration-*"],
```

Comment:
This gets rid of all the changesets noise for packages that aren't published, which would otherwise be made even worse in this PR now that each integration helper template is a workspace package.